_includes/snippets/architecture/01/de.html

```diff
@@ -3,50 +3,56 @@
 {% highlight awk %}
 sass/
 |
+|– abstracts/
+|   |– _variables.scss    # Sass Variablen
+|   |– _functions.scss    # Sass Funktionen
+|   |– _mixins.scss       # Sass Mixins
+|   |– _placeholders.scss # Sass Platzhalter
+|
 |– base/
-|   |– _reset.scss       # Reset/normalize
-|   |– _typography.scss  # Regeln für Typographie
-|   …                    # Etc.
+|   |– _reset.scss        # Reset/normalize
+|   |– _typography.scss   # Regeln für Typographie
+|   …                     # Etc.
 |
 |– components/
-|   |– _buttons.scss     # Buttons
-|   |– _carousel.scss    # Carousel
-|   |– _cover.scss       # Cover
-|   |– _dropdown.scss    # Dropdown
-|   …                    # Etc.
+|   |– _buttons.scss      # Buttons
+|   |– _carousel.scss     # Carousel
+|   |– _cover.scss        # Cover
+|   |– _dropdown.scss     # Dropdown
+|   …                     # Etc.
 |
 |– layout/
-|   |– _navigation.scss  # Navigation
-|   |– _grid.scss        # Grid system
-|   |– _header.scss      # Header
-|   |– _footer.scss      # Footer
-|   |– _sidebar.scss     # Sidebar
-|   |– _forms.scss       # Forms
-|   …                    # Etc.
+|   |– _navigation.scss   # Navigation
+|   |– _grid.scss         # Grid system
+|   |– _header.scss       # Header
+|   |– _footer.scss       # Footer
+|   |– _sidebar.scss      # Sidebar
+|   |– _forms.scss        # Forms
+|   …                     # Etc.
 |
 |– pages/
-|   |– _home.scss        # Home spezifische Styles
-|   |– _contact.scss     # Contact spezifische styles
-|   …                    # Etc.
+|   |– _home.scss         # Home spezifische Styles
+|   |– _contact.scss      # Contact spezifische styles
+|   …                     # Etc.
 |
 |– themes/
-|   |– _theme.scss       # Default-Theme
-|   |– _admin.scss       # Admin-Theme
-|   …                    # Etc.
+|   |– _theme.scss        # Default-Theme
+|   |– _admin.scss        # Admin-Theme
+|   …                     # Etc.
 |
 |– utils/
```

Comment:
Yeah sorry, I guess I got a little confused a some point with the changelog  :see_no_evil: